_infographics/klimaticke-ukazovatele/koncentracia-co2.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout:     infographic
-title:      "Vývoj koncentrácia CO₂ v atmosfére"
+title:      "Vývoj koncentrácie CO₂ v atmosfére"
```

Comment:
@LenkaSedlackova Co ta nakoniec viedlo k tomu dat sem plural? Nebavili sme sa este spolu, ze tam dame singular vsade?

Reply:
veď "koncentrácie" je singulár, nie?

Reply:
No comment, som tupelo, uz sa mi do toho pletie cestina :D.

Reply:
heh to nie si sám, ja mám otvorený slovenský slovník pri prekladoch... a kontrolujem si aj väzby s predložkami (slúžiť na niečo/k niečomu atď) :D